src/plugins/plugin.legend.js

```diff
@@ -90,8 +90,8 @@ defaults._set('global', {
  * @return {Number} width of the color box area
  */
 function getBoxWidth(labelOpts, fontSize) {
-	return labelOpts.usePointStyle ?
-		fontSize * Math.SQRT2 :
+	return labelOpts.usePointStyle && labelOpts.boxWidth > fontSize ?
```

Comment:
Why not simply `return labelOpts.boxWidth,` all the time since we are testing `usePointStyle` in other places?